Wikipedia/Code/WebViewController.m

```diff
@@ -335,6 +336,23 @@ - (void)minimizeFindInPage {
     }
 }
 
+- (CGFloat)marginWidthForSize:(CGSize)size {
```

Comment:
@joewalsh Should this be marginWidthForWidth? Height isn't used and not sure we'd ever need to consider it...